releases/v20.2.0.md

```diff
@@ -0,0 +1,126 @@
+---
+title: What&#39;s New in v20.2.0
+toc: true
+summary: Additions and changes in CockroachDB version v20.2.0.
+---
+
+## XX, 2020
+
+With the release of CockroachDB v20.2, we've made a variety of productivity, performance, and security improvements. Check out a comprehensive [summary of the most significant user-facing changes](#summary) and then [upgrade to CockroachDB v20.2](../v20.2/upgrade-cockroach-version.html). You can also read more about these changes in the [v20.2 blog post - NEED LINK](https://www.cockroachlabs.com/blog/cockroachdb-20-1-release/).
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+## Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+## Docker image
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ docker pull cockroachdb/cockroach:v20.2.0
+~~~
+
+## Summary
+
+This section summarizes the most significant, user-facing changes in v20.2.0. For a complete list of features and changes, including bug fixes and performance improvements, see the [release notes](index.html#testing-releases) for previous testing releases.
+
+{{site.data.alerts.callout_info}}
+"Core" features are freely available in the core version and do not require an enterprise license. "Enterprise" features require an [enterprise license](https://www.cockroachlabs.com/get-cockroachdb/). [CockroachCloud clusters](https://cockroachlabs.cloud/) include all enterprise features.
+{{site.data.alerts.end}}
+
+- [CockroachCloud](#cockroachcloud)
+- [SQL](#sql)
+- [Recovery and I/O](#recovery-and-io)
+- [Deployment and Operations](#deployment-and-operations)
+- [Observability](#observability)
+- [Backward-incompatible changes](#backward-incompatible-changes)
+- [Known limitations](#known-limitations)
+- [Education](#education)
+
+<style>
+    table td:first-child {
+        min-width: 100px !important;
+    }
+    table td:nth-child(2) {
+        min-width: 200px !important;
+    }
+</style>
+
+### CockroachCloud
```

Comment:
@Amruta-Ranade, @kannanlakshmi, I'm not clear on what new CC features should be called out here. Please help.